### PR TITLE
 AzurestackAdmin  TP2 changes

### PR DESCRIPTION
**Pull request by bganapa**
AzurestackAdmin  TP2 changes

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@bganapa__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (bganapa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by markcowl** [comment]
@bganapa Please pull the latest and the CredScan issues should be resolved.

**Comment by bganapa** [comment]
merged the changes for CredScan issues. its green now

**Comment by bganapa** [comment]
@markcowl merged the changes for CredScan issues. its green now

**Comment by hovsepm** [comment]
@bganapa please add tests.

**Comment by bganapa** [comment]
@hovsepm We have tests at the powershell level, not at SDK. whats the general practice?

**Comment by markcowl** [comment]
@bganapa That is the opposite of the advice.  We suggest api testing here and a smaller set of unit tests and scenario tests in powershell

**Comment by hovsepm** [comment]
@bganapa please remove [no tests] prefix from the title when your PR will be ready to merge

**Comment by hovsepm** [comment]
> using System;

please add license header 

---
Refers to: src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/LocationTests.cs:1 in 7d13cef. [](commit_id = 7d13cef6355740f8515ecce4ff1629550c87920c, deletion_comment = False)

**Comment by hovsepm** [comment]
> using System;

please add license header 

---
Refers to: src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/OfferTests.cs:1 in 7d13cef. [](commit_id = 7d13cef6355740f8515ecce4ff1629550c87920c, deletion_comment = False)

**Comment by hovsepm** [comment]
> using System;

please add license header 

---
Refers to: src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/PlanTests.cs:1 in 7d13cef. [](commit_id = 7d13cef6355740f8515ecce4ff1629550c87920c, deletion_comment = False)

File: src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/ResourceProviderManifest.cs [hunk 1]
```diff
@@ -0,0 +1,222 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (hovsepm) [inline src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/ResourceProviderManifest.cs]: please add license header

File: src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/SubscriptionTests.cs [hunk 2]
```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (hovsepm) [inline src/ResourceManagement/AzureStackAdmin/AzureStackAdmin.Tests/SubscriptionTests.cs]: please add license header

**Comment by markcowl** [comment]
Is this related to this PR: https://github.com/Azure/hydra-specs-pr/pull/1309 If so, please address the comments there - we can consider this PR after that one is merged and you're regenerated the client.

**Comment by bganapa** [comment]
@hovsepm  added copy right headers, @markcowl added documentation and generated the clients for this PR

**Pull request merged**